src/components/view/Information.jsx

```diff
@@ -4,26 +4,36 @@ const Information = ({
   dimensions,
   weight,
   manufacturer,
+  ASIN,
   country,
-  unitPrice,
-  totalPrice,
+  modelNum,
+  date,
+  frequency,
 }) => {
   return (
-    <div className="grid grid-cols-2 w-6/12 justify-start divide-y divide-gray-400 font-outfit mb-8">
-      <div className="bg-gray-200 pb-2 px-8 py-2 font-semibold">
-        Product Dimensions
+    <div className="py-6">
+      <div className="font-semibold font-outfit text-2xl my-4">
+        {" "}
+        Product Information
+      </div>
+      <div className="grid grid-cols-2 w-6/12 justify-start divide-y divide-gray-300 font-outfit font-medium mb-8">
+        <div className="bg-gray-200 pb-2 px-8 py-2 ">Product Dimensions</div>
```

Comment:
this can surely be mapped :sob: I would create an object with the display text and the accessor value